File: FormalBench-data/diverse/rule_23/CountOddSquares.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class CountOddSquares {

	public static int countOddSquares(int n, int m) {
		int count = 0;
		for (int i = n; m >= i; i++) {
			double square = Math.sqrt(i);
			if (square % 1 == 0 && i % 1 == 0) {
				count++;
			}
		}
		return count;
	}
}

File: FormalBench-data/FilteredData/fault_mutants/run_ukcw__uc/CountOddSquares/mutants/20/c/CountOddSquares.c
#include "java_arrays.h"
#include <math.h>
#include <stdint.h>

static int32_t java_add(int32_t left, int32_t right) { return (int32_t)((uint32_t)left + (uint32_t)right); }

int32_t countOddSquares(int32_t n, int32_t m) {
    int32_t count = INT32_C(0);
    for (int32_t i = n; (i <= m); i = java_add(i, INT32_C(1))) {
        double square = sqrt(i);
        if ((fmod(square, INT32_C(1)) == INT32_C(0))) {
            count = java_add(count, INT32_C(1));
        }
    }
    return count;
}
